nginx.conf

```diff
@@ -0,0 +1,18 @@
+events {
+  worker_connections 768;
+}
+http {
+  upstream dokomo {
+    server webapp:8888;
```

Comment:
It's some magic involving `/etc/hosts` I think...
https://docs.docker.com/userguide/dockerlinks/